SU-HSDO/suhumsci
# HSD8-283 Removed blocks in favor of menu item

# READY FOR REVIEW

# Summary
- Removed blocks in favor of menu item

# Needed By (Date)
- End of sprint, sept 27

# Urgency
- low

# Steps to Test
1. checkout this branch
1. `composer install --prefer-source`
1. `blt drupal:sync --site=archaeology`
1. verify the masquerade block on the top of all admin pages is gone
1. verify a new menu item exists "Masquerade" under the "People" menu item in the admin toolbar.
1. navigate to that menu item
1. masquerade as "Howard"
1. verify masquerading works as expected.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
